src/components/color/examples/ColorExamples.tsx

```diff
@@ -1,24 +1,41 @@
 import * as React from 'react';
 import * as _ from 'underscore';
-import {Tooltip} from '../../tooltip/Tooltip';
 import {Color} from '../Color';
 
 export class ColorExamples extends React.Component {
     render() {
         const VaporColors: {[key: string]: string} = require('!sass-variable-loader?preserveVariableNames!coveo-styleguide/scss/common/palette.scss');
         const colors = _.map(VaporColors, (value, color) => (
-            <Tooltip title={`${color}: ${value.toUpperCase()}`} placement='top' className='spaced-box'>
-                <Color
-                    key={color}
-                    color={color}
-                    className='p2 text-pure-white material-card with-hover'
-                />
-            </Tooltip>
+            (value === "'category'") ? (

```

Comment:
No need for the parenthesis or the double quotes.
```suggestion
            value === 'category' ? (
```

Reply:
The double quote are necessary because the outputed string is actually 'category' (with the single quote)